README.md

```diff
@@ -49,9 +49,9 @@ If *description* is given it will be used as the *short_description* value on th
 The *list* module provides methods to create extra fields for use in the list view and should be used within a **admin.ModelAdmin** declaration
 
 #### short_text(field_name, length=200, name='', description='', suffix='...', strip_html=False)
-Returns a function to be used as a *list_display* entry.  
-If no name is given, *field_name* is used. 
-You may change the suffix which will be appended to **shortened** text only.  
+Returns a function to be used as a *list_display* entry.
+If no name is given, *field_name* is used.
+You may change the suffix which will be appended to **shortened** text only.
```

Comment:
Pleas revert all those trailing double spaces. In Git Markdown it represents a line break